test/Garnet.test/RespListTests.cs

```diff
@@ -1440,5 +1440,122 @@ public void CheckListOperationsOnWrongTypeObjectSE()
             // LMPOP RIGHT
             RespTestsUtils.CheckCommandOnWrongTypeObjectSE(() => db.ListRightPop(keys, 3));
         }
+
+        #region LPOS
+
+        [Test]
+        [TestCase("a,c,b,c,d", "a", 0)]
+        [TestCase("a,c,b,c,adc", "adc", 4)]
+        [TestCase("a,c,b,c,d", "c", 1)]
+        [TestCase("av,123,bs,c,d", "e", null)]
+        public void LPOSWithoutOptions(string items, string find, int? expectedIndex)
+        {
+            using var redis = ConnectionMultiplexer.Connect(TestUtils.GetConfig());
+            var db = redis.GetDatabase(0);
+            var key = "KeyA";
+            string[] arguments = [key, .. items.Split(",")];
+
+            db.Execute("RPUSH", arguments);
+
+            var actualIndex = (int?)db.Execute("LPOS", key, find);
```

Comment:
May be, the ListPosition(s) API could be used.

Reply:
Personally, I want to test every combination in the raw input like case difference, different option order or without default values but with ListPosition(s) I can't test these. I have added a separate test to test with ListPosition(s) API. Hope that's fine.